ansible/roles/demo-certs/tasks/install-certs.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: install_certs - Copy CA Certs
+  tags:
+    - install_certs
+  copy:
+    src: tls/ca/ca.crt
+    dest: "{{ redpanda_truststore_file }}"
+    owner: redpanda
+    group: redpanda
+    force: "{{ overwrite_certs | default('no') | bool }}"
+
+- name: install_certs - Copy Node Certs
+  tags:
+    - install_certs
+  copy:
+    src: tls/certs/{{ansible_hostname}}/node.crt
+    dest: "{{ redpanda_cert_file }}"
+    owner: redpanda
+    group: redpanda
+    force: "{{ overwrite_certs | default('no') | bool }}"
+
+- name: install_certs - Set Redpanda config
+  tags:
+    - install_certs
+  set_fact:
+    tls: true
+
+- name: install_certs - Apply updated config via reapplying the redpanda_broker role
+  tags: install_certs
+  import_role:
+    name: redpanda_broker
+  when: not skip_node | default(false) | bool
```

Comment:
There's a lot of nasty stuff that happens with dependencies in playbooks so I'd prefer to avoid explicit ones unless there's a really serious advantage. More info on some imports that caused bugs [in this commit ](https://github.com/redpanda-data/deployment-automation/pull/159/commits/520208224ab09aa939394d799fd4bf60a77b2093)on the linter PR